edweiss-app/model/quizzes.ts

```diff
@@ -38,7 +38,7 @@ namespace Quizzes {
         showResultToStudents: boolean;
         // inherits:
         // name: string
-        // type: AssignmentType = "quiz"|"submit"
+        // type: AssignmentType = "quiz"|"submission"
         // dueDate: TimeStamp
```

Comment:
Quizzes are not my part. I think Ferdinand works on it, I prefer let him refractor his stuff. I just decided to change the AssignmentType everywhere in the code to make it be "submission" instead of "submit" which is much better for understanding.